src/main/java/net/masterthought/cucumber/TagObject.java

```diff
@@ -143,6 +147,18 @@ public int getNumberOfPending() {
         return statuses;
     }
 
+    private List<Status> getStatusesByExcludingBackGround() {
+        List<Status> statuses = new ArrayList<Status>();
+        for (ScenarioTag scenarioTag : scenarios) {
+            if (!scenarioTag.getScenario().isBackground() && scenarioTag.hasSteps()) {
```

Comment:
Make double check whether you need additional checking such as

    scenarioTag.hasSteps()


because I believe that if there is no scenario at all

    scenarioTag.getScenario().getSteps()

simple returns empty list and the loop will be executed 0 times